src/test/java/plugins/WorkflowPluginTest.java

```diff
@@ -135,6 +137,8 @@
         build.shouldContainsConsoleOutput("Building version 1.0-SNAPSHOT");
         jenkins.restart();
         // Default 120s timeout of Build.waitUntilFinished sometimes expires waiting for RetentionStrategy.Always to tick (after initial failure of CommandLauncher.launch: EOFException: unexpected stream termination):
+        System.out.println("Waiting for slave to come online...");
+        Thread.sleep(10000); //Jenkins needs some extra time to wake up before we can query the api
```

Comment:
Not sure what this refers to. Is this purporting the address the comment on line 139?